static-build/make_package.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh -x
+
+# This script helps make static built package for tarantool with CPack
+# in docker container. Uses packpack docker image for building. Usaly
+# is runned by CI/CD system. For run this script you need to set variables:
+# VERSION - version of tarantool
+# CMAKE_TARANTOOL_ARGS - cmake args for tarantool
+# Variable VERSION is required for making names of packages. All packages
+# could be found in ../build/ folder.
+
+USER_ID=$(id -u)
+PACKAGE_DIR="${PACKAGE_DIR:-build}"
+
+# Default values for variables for success  run script on local machine.
+VERSION="${VERSION:-0.0.1}"
+CMAKE_TARANTOOL_ARGS="${CMAKE_TARANTOOL_ARGS:--DLUAJIT_ENABLE_GC64=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo}"
+
+echo "User id: ${USER_ID}"
+echo "Build version: ${VERSION}"
+echo "CMake args: ${CMAKE_TARANTOOL_ARGS}"
+
+# Run building in docker container with right user id for correct 
+# permissions of artifacts.
+if [ "${USER_ID}" = "0" ]; then
+    docker run --rm \
+        --env VERSION=${VERSION} \
+        --env PACKAGE_DIR=${PACKAGE_DIR} \
+        --volume $(pwd):/tarantool \
+        --workdir /tarantool/static-build/ \
+        packpack/packpack:centos-7 sh -c "
```

Comment:
Please move this to the line with `--volume $(pwd):/tarantool` like you do at L39.